docs/GraphOperationAPIs.md

```diff
@@ -210,7 +210,7 @@ cdef extern from "[operation header]" namespace "[namespace]" nogil:
             [...]
 
         _[Operation]Plan()
-        _[Operation]Plan(const PropertyFileGraph * pfg)
```

Comment:
In the Cython code in this documentation it needs to be `_PropertyGraph*` not `PropertyGraph*`. So where ever if was `PropertyFileGraph` use the `_`.